src/main/resources/runner/docker/haskell/Dockerfile

```
@@ -1,12 +1,16 @@
 FROM haskell:8
-# TODO FIND A SMALLER BASE IMAGE TO RUN ON ALPINE
-# TODO ADD ANY OTHER DEPENDENCIES
 
 ARG DIR
+ARG USER=jeff # Consider making this an env variable

```

Just to stay consistent
```suggestion
ARG USER=appuser
```